src/app/home/home.page.html

```diff
@@ -4,42 +4,51 @@
       Welcome {{userName}}
     </ion-title>
     <ion-buttons slot="end" (click)="onLogout()">
-      <ion-img class="logout-icon" src="assets/images/log-out.svg">
+      <ion-img class="logout-icon" src="assets/images/logout_6.png">
       </ion-img>
     </ion-buttons>
   </ion-toolbar>
 </ion-header>
 
 <ion-content class="page">
-  <ion-slides class="media-slides" [options]="sliderOpts" #slider>
-    <ion-slide *ngFor="let img of sliderImages; let i = index">
-      <ion-img class="slider-img" [src]="img"></ion-img>
-    </ion-slide>
-  </ion-slides>
+  <ion-item *ngIf="!hardwareSoftwareAvailability.appAvailability" lines="none" class="top-msg">
+    <div class="top-msg-styles">  Dear user, please install and use
+      <span class="app-name-link" (click)="openPlayStore()">
+        Aarogya Setu App</span> for the tracking of covid 19</div>
+  </ion-item>
+  <div class="settings-heading">Device Settings</div>
+  <div class="device-settings-container" no-lines>
+    <ion-item lines="none" class="settings-items">
+      <ion-label class="settings-label">Location Status</ion-label>
+        <div *ngIf="hardwareSoftwareAvailability.locationStatus">ON</div>
```

Comment:
{{ hardwareSoftwareAvailability.locationStatus ? "ON" : "OFF"}}
Can we do like this